app/leagues/page.tsx

```diff
@@ -11,9 +12,9 @@ import { LeagueCard } from '@/components/LeagueCard/LeagueCard';
 const Leagues = (): JSX.Element => {
   return (
     <div className="Leagues mx-auto max-w-3xl pt-10">
-      <h1 className="pb-10 text-center text-3xl font-bold tracking-tight">
+      <Heading heading="h2" className="pb-10 text-center">
```

Comment:
I'd change this back to what it was prior so that we can make another ticket to replace all existing headers with the new `Heading` component upon approval

Reply:
Removed the changes from this file.